CHANGELOG.md

```diff
@@ -394,3 +394,7 @@ prevented strings passed first to the text-rendering methods to be displayed.
 * turned `accept_page_break` into a property
 * unit tests now use the standard `unittest` lib
 * massive code cleanup using `flake8`
+
+## [2.5.7] - not released yet
```

Comment:
This should be at the top of the `CHANGELOG.md` file :)
You may have to rebase your branch because I just merged PR #492 that added an entry to this file too